ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -21,16 +21,28 @@ import Foundation
 public enum CocoaPodUtils {
   // MARK: - Public API
 
-  /// Information associated with an installed pod.
-  public struct PodInfo {
+  public struct VersionedPod {
     /// Public name of the pod.
-    var name: String
+    let name: String
 
     /// The version of the pod.
-    var version: String
+    let version: String
+  }
+
+  /// Information associated with an installed pod.
+  public struct PodInfo {
+    let versionedPod: VersionedPod
 
     /// The location of the pod on disk.
     var installedLocation: URL
+
+    func name() -> String {
```

Comment:
Nit: It'd be nice to have these as properties instead of functions with `lazy`:

```
lazy var name: String { return self.versionedPod.name }
lazy var version: String { return self.versionedPod.version }
```

Essentially identical but no need for the `()`

Reply:
But still an improvement to add the property :)